### ISSUE TITLE
Graph of recombination events algorithm 

### ISSUE THREAD

**Issue by valery-shap** [jameshadfield/phandango#142]
Hello,

I have the data from gubbins gff file with coordinated (Start and Stop, and snp_count), I've downloaded this data and had the graph with peaks at the bottom. Could you please explain what is y-value? and how this value was calculated? I've tried to do the same distribution and had different result)

Best regards,
Valery

**Comment by jameshadfield** [comment]
Hi Valery, the y-axis is the number of recombination events observed which cover that particular position in the genome. Note that an event may involve many strains but only be counted once (the red blocks indicate events which are inferred on ancestral nodes and therefore involve more than one tip).

**Comment by valery-shap** [comment]
sorry, one more clarifying question. by the number of recombination events Do you mean exactly the number of strains that were detected in the regions of recombination on that position? not the mean number of snps.

**Comment by jameshadfield** [comment]
> Do you mean exactly the number of strains that were detected in the regions of recombination on that position?

No, the number of recombination events. If a recombination event is inferred to happen at an ancestral node then one event will involve multiple strains. The Gubbins paper details how these are inferred, phandango just visualises them.

> I supposed that y-axis is the mean number of snps, that was counted:

No. It is the number of recombination events.

**Comment by valery-shap** [comment]
Ok, thank you, I'll try, it's special terminology from gubbins. So if we return to the output gff file and the output of Phandango graph. Does it count exactly number of rows where the value from the last column "node=" is the ancestral node?
